### ISSUE TITLE
NFC menu is missing in Android 9

### ISSUE THREAD

**Issue by yosepito** [NightscoutFoundation/xDrip#1334]
I assume it's an issue of android 9. Under the same version of Xdrip, my old android 6 shows the NFC menu right below "source of data" (the device has no NFC hardware), while my new android 9 does not show it (and to my surprise, it does actually have NFC hardware).

Strange behaviour.

How can I scan with NFC or even start a new sensor?

**Comment by Navid200** [comment]
Please post in support channels.
https://www.facebook.com/groups/xDripG5
https://gitter.im/jamorham/xDrip-plus
There may be other users who have experienced the same.

**Comment by yosepito** [comment]
@Navid200 
Sorry but that was not me. I don't have facebook. This is was solved somehow. Don't know how, but after activating the NFC in the smartphone I had to select the "wallet". In options of NFC I had 2 possibilities, HCE and the SIM card. I selected HCE and then it worked. I can't remember whether a restart was needed, but in such case I recommend doing it.
I'm not a user of G5, so I don't know what you referred with the links above, sorry.
